examples/configs/generate.cfg

```diff
@@ -46,11 +46,11 @@ verbose = 0
 # set CSV column name, timestep duration in sec. (example: 10 min) and factor (example: input in W)
 # include_ext_load_csv_option = [["column", "value"], ["step_duration_s", 600], ["factor", 0.001]]
```

Comment:
In all cfg files: Change `ext load` to `fixed load`

Reply:
done